spec/support/acceptance/servicenow/mock_instance.rb

```diff
@@ -20,13 +20,11 @@ class MockServiceNowInstance < Sinatra::Base
       SSLEnable: true,
       SSLCertName: [['CN', 'Sinatra']]
 
-  # Handles authorization
-  use Rack::Auth::Basic do |user, password|
-    user == 'mock_user' && password == 'mock_password'
-  end
-
   before do
     content_type 'application/json'
+    unless ['Basic bW9ja191c2VyOm1vY2tfcGFzc3dvcmQ=', 'Bearer mock_token'].include?(env['HTTP_AUTHORIZATION'])
```

Comment:
We should also check `user/password` here too to make sure those credentials are also right. Logic should be "if both user/password and oauth included, fail. Otherwise check relevant credentials".

Reply:
The `Basic bW9ja191c2VyOm1vY2tfcGFzc3dvcmQ=` string, is the user/password check. Username/Password gets Base64 encoded and put in the `HTTP_AUTHORIZATION` env value.

Reply:
Oh ok, my bad for not seeing that. Cool.